README.md

```
@@ -20,9 +20,13 @@ Since the manual mode does not have strong dependencies on any platform (i.e., p
 
 - Create a workdir for the installer - `mkdir sno-workdir`
 - Create an `install-config.yaml` in the sno-workdir. An example file can be found in `./install-config.yaml.template`. There are a small number of fields in the template that should be set by hand. Reasonable defaults are given below.
-    * `MACHINE_NETWORK` - the machine network CIDR. A good default is `192.168.126.0/24`.
-    * `CLUSTER_SVC_NETWORK` - the cluster service network CIDR. A good default is `172.30.0.0/16`.
-    * `CLUSTER_NETWORK` - the cluster network CIDR. A good default is `10.128.0.0/14`.
+    * `IP_STACK` - one of `v4`, `v6`, `v4v6` (dual-stack primary v4), `v6v4` (dual-stack primary v6). **In dual-stack, the primary family is always listed first in `install-config.yaml`**.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)